services/website/website-redirect.service.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const { escapeFormat } = require('../../core/badge-urls/path-helpers')
+const { redirector } = require('..')
+
+function escapeFormatSlashes(t) {
+  return (
+    escapeFormat(t)
+      // Double slash
+      .replace(/\/\//g, '/')
+  )
+}
+
+/*
+Old documentation, for reference:
+
+<p>
+  <code>[OPTIONS]</code> can be:
+  <ul>
+    <li>
+      Nothing:&nbsp;
+      <code>…/website/…</code>
+    </li>
+    <li>
+      Online and offline text:&nbsp;
+      <code>…/website-up-down/…</code>
+    </li>
+    <li>
+      Online and offline text, then online and offline colors:&nbsp;
+      <code>…/website-up-down-green-orange/…</code></li>
```

Comment:
I created redirects for the old URL pattern. The new one matches what we did for Nodeping, and can be fully customized in the badge modal.